Lib/test/test_codecs.py

```diff
@@ -875,95 +875,6 @@ def test_surrogatepass_handler(self):
             b"abc\xed\xa0z".decode(self.encoding, "surrogatepass")
 
 
-@unittest.skipUnless(sys.platform == 'win32',
-                     'cp65001 is a Windows-only codec')
-class CP65001Test(ReadTest, unittest.TestCase):
```

Comment:
Does utf-8 pass these tests?

Reply:
I confirmed utf-8 passes this test.

Reply:
> Does utf-8 pass these tests?

Sorry, I forgot to specify before I merged my PR that yes: I tested on my Windows 10 and the test still passed. But CP65001Test is now redundant with UTF8Ttest.

> I confirmed utf-8 passes this test.

Thanks for checking :-)